drizzlepac/haputils/make_poller_files.py

```diff
@@ -0,0 +1,154 @@
+    Nothing.
+    """
+    # Open rootname list file
+    f = open(input_list, 'r')
+    rootname_list = f.readlines()
+    f.close()
+    output_list = []
+    for rootname in rootname_list:
+        rootname = rootname.strip()
+        fullfilepath = locate_fitspath_from_rootname(rootname)
+        if len(fullfilepath) > 0:
+            print("Rootname {}: Found fits file {}".format(rootname, fullfilepath))
+            imgname = fullfilepath.split("/")[-1]
+        else:
+            # Warn user if no fits file can be located for a given rootname, and skip processing of the file.
+            print("WARNING: No fits file found for rootname '{}'. This rootname will be omitted from poller "
+                  "file generation.".format(rootname))
+            continue
+        # Build each individual poller file line
+        linelist = []
+        linelist.append(imgname)
+        imghdu = fits.open(imgname)
+        imghdr = imghdu[0].header
+        linelist.append("{}".format(imghdr['proposid']))
+        linelist.append(imgname[1:4].upper())
+        linelist.append(imghdr['linenum'].split(".")[0])
+        linelist.append("{}".format(imghdr['exptime']))
+        if imghdr['INSTRUME'].lower() == "acs":
+            filter = poller_utils.determine_filter_name("{};{}".format(imghdr['FILTER1'], imghdr['FILTER2']))
+        elif imghdr['INSTRUME'].lower() == "wfc3":
+            filter = poller_utils.determine_filter_name(imghdr['FILTER'])
+        linelist.append(filter.upper())
+        linelist.append(imghdr['detector'].upper())
+        if poller_file_type == 'mvm':  # Additional stuff to add to MVM poller files
+            linelist.append("skycell-{}".format(skycell_name))
+            linelist.append("NEW")
+        linelist.append(fullfilepath)
+        imghdu.close()
+        # Append newly created poller file line to the list of lines to be written to the output file.
+        output_list.append(",".join(linelist))
+    # adding carriage returns to all but the very last line in the output file.
+    list_size = len(output_list)
+    for ctr in range(0, list_size):
+        if ctr != list_size-1:
+            trailing_char = "\n"
+        else:
+            trailing_char = ""
+        output_list[ctr] = output_list[ctr]+trailing_char
+
+    # write output poller file
+    with open(output_poller_filename, 'w') as f:
+        f.writelines(output_list)
+    print("wrote {} poller file '{}'.".format(poller_file_type.upper(), output_poller_filename))
+
+
+# ============================================================================================================
+
+def locate_fitspath_from_rootname(rootname):
+    """returns full file name (fullpath + filename) for a specified rootname.
+
+    Parameters
+    ----------
+    rootname : str
+        rootname to process
+
+    Returns
+    -------
+    fullfilepath : str
+        full path + image name of specified rootname.
+    """
+    if in_args.master_observations_file is "":
```

Comment:
Something is not right here, as `in_args.master_observations_file` gets defined in `search_skyframes.py` not this file, so this check will never succeed.  Perhaps, just checking that 'DATA_PATH' is defined and exists is what was intended.

Reply:
Issue resolved by commit [d584864](https://github.com/spacetelescope/drizzlepac/pull/964/commits/d584864cbce664123ea42889659c6cf656bfba3a)